README.md

```diff
@@ -109,6 +103,27 @@ Whether `package.json` and `yarn.lock` are in the current directory.
 | `fzf-make --help` / `fzf-make help`                       | Show help                                     |
 | `fzf-make --version` / `fzf-make -v` / `fzf-make version` | Show version                                  |
 
+## Example Aliases
+To simplify the usage of `fzf-make`, you can define aliases in your shell configuration. Below is an example configuration that works for most shells (bash, zsh, fish, etc.):
+
+```sh
+alias fm='fzf-make'
+alias fr='fzf-make repeat'
+alias fh='fzf-make history'
+```
+### Setting Up
+1. Add the above lines to your shell's configuration file:
+* For bash: `~/.bashrc`
+* For zsh: `~/.zshrc`
+* For fish: `~/.config/fish/config.fish` (use alias syntax as shown above)
+1. Reload your shell configuration:
+bash/zsh: `source ~/.bashrc` or `source ~/.zshrc`
+fish: `source ~/.config/fish/config.fish`
+### Example Usage
+* `fm`: Launch fzf-make to select and run a target.
+* `fr`: Repeat the last executed command.
+* `fh`: Open fzf-make with the history pane focused.
```

Comment:
I think it's enough that showing example settings only. 👍
So please delete L114-L125.

```suggestion
```

Reply:
fixed